lammps-tutorials.tex

```diff
@@ -677,46 +631,67 @@ \subsubsection{My first input}
 $\sigma_{12} = \sqrt{1.0 \times 3.0} = 1.732$.
 
-\paragraph{Energy minimization}
+\paragraph{Single point energy}
 The system is now fully parameterized and the input ready to compute
 forces.  Let us fill in the two last remaining categories,
 \textit{Visualization}, and \textit{Run}, by adding the following lines
-into \textit{input.lmp}:
+into \textit{initial.lmp}:
 {\normalsize
 \begin{verbatim}
- # 4) Visualization
- thermo 10
- thermo_style custom step etotal press
+# 4) Visualization
+thermo 10
+thermo_style custom step etotal press
 
- # 5) Run
- minimize 1.0e-6 1.0e-6 1000 10000
+# 5) Run
+run 0 post no
 \end{verbatim}
 }
 The \textit{thermo} command asks LAMMPS to print thermodynamic
-information to the console every given number of steps, in this
-input every 10 simulation steps.  The \textit{thermo\_style custom} determines
-\emph{which} output LAMMPS should print.  This selection prints the
-the step number (\textit{step}), the total energy (\textit{etotal}), and the
-pressure (\textit{press}).
-
-Finally, the \textit{minimize} command tells LAMMPS to perform an energy
-minimization of the system.  That is, LAMMPS will compute the forces on
-all atoms and then update the positions according to a selected
-algorithm so that the potential energy will decrease.  By default,
-LAMMPS uses the conjugate gradient (CG) algorithm
-\cite{hestenes1952methods}.  The simulation will stop as soon at the
-minimizer algorithm cannot find a way to to lower the potential energy.
-Except for trivial systems, minimization algorithms will find the
-``next'' minimum, which may not be the global minimum.
-
-Now you can run the simulation by selecting \textit{Run LAMMPS from Editor
-  Buffer} from the \textit{Run} menu, or by using the
-\texttt{Ctrl-Enter} keyboard shortcut or by clicking on the green ``Run
-LAMMPS'' button (second from left) in the status bar.  The simulation
-can be stopped at any time by using \textit{Stop LAMMPS} from the
-\textit{Run} menu, the \texttt{Ctrl-/} keyboard shortcut or by clicking
-on the red stop button in the status bar.  A new run will always
-``forget'' the current system and start over from the top.
+information to the console every given number of steps, in this input
+every 10 simulation steps.  The \textit{thermo\_style custom} determines
+\emph{which} output LAMMPS should print.  This selection prints the the
+step number (\textit{step}), the total energy (\textit{etotal}), and the
+pressure (\textit{press}).  The \textit{run 0} command will tell LAMMPS
+just to a force and energy initialization.
+
+You can now run LAMMPS.  It should finish quickly and with the default
+settings LAMMPS--GUI will open two new windows, one with the console
+output and one with a chart.  The output window will show some info
+printed by the various command in the input as they are executed and
+the total energy and pressure at step 0, the requested thermodynamic
+data.  Since we have not done any real simulation steps, the charts
```

Comment:
```suggestion
the total energy and pressure at step 0, i.e. the requested thermodynamic
data.  Since no real simulation steps have been performed, the charts
```